lerna.json

```diff
@@ -5,6 +5,7 @@
     "apps/*",
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30514&commitId=9b3b30458df65d7e9ccd5cb864f8bc8763b73fb0&env=prod&clientType=FLUENTUI)


  <details><summary><b>Divider</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider.Divider Vertical Example Shorthand.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>